### ISSUE TITLE
MenuItem background selector doesn't have same color

### ISSUE THREAD

**Issue by trietbui85** [negusoft/holoaccent#27]
I test this app in 2 devices: Nexus 7 2013 (Android 4.4.2) and Galaxy S3 (Android 4.3.1) and see its MenuItem background is different. I choose theme `Light Colored ActionBar Inverse`
- Nexus 7 2013:
  ![image](https://cloud.githubusercontent.com/assets/340836/3269244/eeba2cc2-f2e9-11e3-9b64-5ffd4cf1ac02.png)
- Galaxy S3:
  ![image](https://cloud.githubusercontent.com/assets/340836/3269277/4d1e59aa-f2ea-11e3-86a3-d06186e9451a.png)

You will notice in Nexus 7 2013, when selected `Dialog Fragment` MenuItem, its background turns to green, while in Galaxy S3 is default holo color.

Do you know why?

BTW, do you also notice although I choose `Light Colored ActionBar Inverse` but it seems MenuItem in Nexus 7 is Dark while in Galaxy S3 is Light theme. Is it expected behavior?

**Comment by blurkidi** [comment]
Indeed, it is a known issue. The list selector for the menu is not customized when it comes from pressing a hardware menu button. It is styled when it is the action bar's overflow menu though.

I spent quite a lot of time trying to figure out how I could customize this selector. I went all the way to the android source and I tried every trick I know to make it work but I am afraid it is just not possible to do so.

I just hope that somebody proofs me wrong...
